2nd Upload of the PB Bratan!

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
I've added the PB Bratan! a CIV class ship thats part nightclub, part cafeteria, part hydroponics, part drug lab and smuggling ship, all in one! 

It's practically a moving POI, and it drives like one. 

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
It's got zero ship weaponry and absolutely abysmal speed and turning. 
It attempts to compensate for it by having a decent- for a civ ship- crew armory. 

they have a lethal and non lethal armory

Non Lethal:
3x Disablers
3x Stun batons

Less Lethal:
2x Kammerers (beanbags)
1x Double Barrel Shotgun (beanbags)
3x Baseball bat 
2x Beanbag shell box
1x buckshot shell box


for armory, they have 2x armor vest and combat helmets, as well as 2x riot armor and 2x riot helmet, + 2x riot shields

## How to test
<!-- Describe the way it can be tested -->
load the .yml file in the IDE, should work! 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
(NOT UPDATED PICTURES)
<img width="899" height="803" alt="bratan live" src="https://github.com/user-attachments/assets/5de9387b-5b09-4371-b017-71f0b3feb915" />
<img width="854" height="800" alt="bratan" src="https://github.com/user-attachments/assets/d787d661-9231-4ef8-8464-05b0736275da" />
<img width="973" height="813" alt="bratan north" src="https://github.com/user-attachments/assets/c211a7d8-d290-4b63-896a-86e1ccb4b5ea" />
<img width="932" height="822" alt="bratan south" src="https://github.com/user-attachments/assets/b2334229-f0a7-4e63-8982-16cfc3004067" />
<img width="695" height="498" alt="bratan VIP room" src="https://github.com/user-attachments/assets/e32c3562-5379-4d35-bc18-4e433be4b33f" />
<img width="428" height="487" alt="vault room 2" src="https://github.com/user-attachments/assets/3c770856-1854-4840-87ee-ffdc513f8621" />



## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: the PB Bratan, a CIV service megaship with a hidden drug lab

Rubeebeebee committed:
2nd upload of the PB bratan!

Rubeebeebee commented:
reuploaded from this:
https://github.com/Monolith-Station/Monolith/pull/1512
because i couldnt figure out how to remove the file that accidentally removes the COG-Arkansaw. Oopsie. 

@OneZerooo0

monolith8319 committed:
chore: automatically update REUSE headers

pheenty commented:
> tweak: added an AME, removed plasma gen, and edited the armory!

Added it to what
The ship isn't in the game yet

Also in case you missed it, https://github.com/Monolith-Station/Monolith/pull/1512#issuecomment-3110158978

Rubeebeebee commented:
> > tweak: added an AME, removed plasma gen, and edited the armory!
> 
> Added it to what The ship isn't in the game yet
> 
> Also in case you missed it, [#1512 (comment)](https://github.com/Monolith-Station/Monolith/pull/1512#issuecomment-3110158978)

asked the guy who commisioned it, they want the name to stay as PB Bratan

Redrover1760 reviewed (changes requested):
Changelog should be for player facing changes, remove the changes you made to the ship you are adding to the changelog

Rubeebeebee commented:
> Changelog should be for player facing changes, remove the changes you made to the ship you are adding to the changelog

gotcha! i'll change them! thank you.

Edit: I've changed them!

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/bratan.yml:
```diff
@@ -0,0 +1,66 @@
+  - Botany
+  engine:
+  - AME
+  - Solar
```
```  
addComponents:
  - type: SolarPoweredGrid
    doNotCull: true
```

Redrover1760 replied:
You'll need to tell the SolarPoweredGrid component to not cull or else your solars will be wonky

Rubeebeebee replied:
there we go! ive added the lines you asked me to and fixed the AME to be on and fueled on roundstart.

Redrover1760 reviewed (changes requested):
AME doesn't start on so the ship sorta just runs out of power within 5 minutes

Rubeebeebee committed:
added
addComponents:
  - type: SolarPoweredGrid
    doNotCull: true

and made sure that the AME is powered on

Rubeebeebee committed:
Merge branch 'bratan-2nd-upload' of https://github.com/Rubeebeebee/Monolith into bratan-2nd-upload

Redrover1760 committed:
Update bratan.yml

Redrover1760 committed:
Merge branch 'main' of https://github.com/Monolith-Station/Monolith into pr/Rubeebeebee/1641

Redrover1760 reviewed (changes requested):
<img width="1127" height="760" alt="Powernet no workie" src="https://github.com/user-attachments/assets/49bf0149-559f-4cc8-a62e-b597355fa1d3" />

your solars powernet does not work

just skips the SMES entirely lol

Redrover1760 commented:
doesn't seem to generate any power for me either in general so no clue whats up with that. Your HV wiring is kinda bad in general.

Rubeebeebee committed:
fixed the solars SMES by adding a connection terminal

Rubeebeebee committed:
Merge branch 'bratan-2nd-upload' of https://github.com/Rubeebeebee/Monolith into bratan-2nd-upload

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> <img alt="Powernet no workie" width="1127" height="760" src="https://private-user-images.githubusercontent.com/39284090/473749331-49bf0149-559f-4cc8-a62e-b597355fa1d3.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTc5NTYsIm5iZiI6MTc1NDE5NzY1NiwicGF0aCI6Ii8zOTI4NDA5MC80NzM3NDkzMzEtNDliZjAxNDktNTU5Zi00Y2M4LWE2MmUtYjU5NzM1NWZhMWQzLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODAzVDA1MDczNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwODBkNjcwNGIyMzhkYjAzMTljN2M4ODBlOWFlZjZkY2VkMjRjN2MyMDRjOWVlNTQ0NjgwMzc4OTU3ZWE1YjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GtSRyuh-fKJg4_9o8K9jRlMfJAXyTxDVbUUvmikY3uE">
> 
> your solars powernet does not work
> 
> just skips the SMES entirely lol

right so! I've fixed the solars SMES by adding a connector terminal. 

The Solars and AME should also feed into the Main SMES line in the engine room as well (engine room is located above the service hallway)
<img width="1091" height="827" alt="bratan HV lines" src="https://github.com/user-attachments/assets/d5b8749f-0edd-41ea-9036-c829f9a873f6" />
<img width="798" height="591" alt="bratan solar term" src="https://github.com/user-attachments/assets/cfb9b43c-1a8c-4ed8-85b6-93b3a97e7f69" />

Redrover1760 commented:
> > <img alt="Powernet no workie" width="1127" height="760" src="https://private-user-images.githubusercontent.com/39284090/473749331-49bf0149-559f-4cc8-a62e-b597355fa1d3.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTc5NTYsIm5iZiI6MTc1NDE5NzY1NiwicGF0aCI6Ii8zOTI4NDA5MC80NzM3NDkzMzEtNDliZjAxNDktNTU5Zi00Y2M4LWE2MmUtYjU5NzM1NWZhMWQzLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODAzVDA1MDczNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwODBkNjcwNGIyMzhkYjAzMTljN2M4ODBlOWFlZjZkY2VkMjRjN2MyMDRjOWVlNTQ0NjgwMzc4OTU3ZWE1YjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GtSRyuh-fKJg4_9o8K9jRlMfJAXyTxDVbUUvmikY3uE">
> > your solars powernet does not work
> > just skips the SMES entirely lol
> 
> right so! I've fixed the solars SMES by adding a connector terminal.
> 
> The Solars and AME should also feed into the Main SMES line in the engine room as well (engine room is located above the service hallway) <img alt="bratan HV lines" width="1091" height="827" src="https://private-user-images.githubusercontent.com/220646896/473750490-d5b8749f-0edd-41ea-9036-c829f9a873f6.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTkzNjAsIm5iZiI6MTc1NDE5OTA2MCwicGF0aCI6Ii8yMjA2NDY4OTYvNDczNzUwNDkwLWQ1Yjg3NDlmLTBlZGQtNDFlYS05MDM2LWM4MjlmOWE4NzNmNi5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwM1QwNTMxMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NzIzZDViNGMzMjRkY2E0YjBjYmRhNzAxZDc1ZTUzYjdkNTEyMGZmNDcwZmQ1NjEyYjc5MzlkMTI1NThjMmI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QOzsjlyciSCh-0l5JWWFT6Y6seta65ELcB72HfRrA48"> <img alt="bratan solar term" width="798" height="591" src="https://private-user-images.githubusercontent.com/220646896/473750491-cfb9b43c-1a8c-4ed8-85b6-93b3a97e7f69.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTkzNjAsIm5iZiI6MTc1NDE5OTA2MCwicGF0aCI6Ii8yMjA2NDY4OTYvNDczNzUwNDkxLWNmYjliNDNjLTFhOGMtNGVkOC04NWI2LTkzYjNhOTdlN2Y2OS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwM1QwNTMxMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYTllY2JhZmMyYjFhZDg0MTVhNDE0YjBhMWViOWM5NDAwODAxYWFmYTIzNmE2ZDgzZjQ1M2U3NTRhMzM1M2I2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fqgpr0ebf-uv7EgA2oNSWqQADVzp8-MQwbdv5OiozVE">

Your other smes still has two cable terminals please review your hv line

Mv line too now that I look at it

Rubeebeebee committed:
fixed the HV and MV lines, added a note for the primiary SMES having a second terminal

Rubeebeebee committed:
Merge branch 'bratan-2nd-upload' of https://github.com/Rubeebeebee/Monolith into bratan-2nd-upload

Rubeebeebee commented:
> > > <img alt="Powernet no workie" width="1127" height="760" src="https://private-user-images.githubusercontent.com/39284090/473749331-49bf0149-559f-4cc8-a62e-b597355fa1d3.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTc5NTYsIm5iZiI6MTc1NDE5NzY1NiwicGF0aCI6Ii8zOTI4NDA5MC80NzM3NDkzMzEtNDliZjAxNDktNTU5Zi00Y2M4LWE2MmUtYjU5NzM1NWZhMWQzLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODAzVDA1MDczNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwODBkNjcwNGIyMzhkYjAzMTljN2M4ODBlOWFlZjZkY2VkMjRjN2MyMDRjOWVlNTQ0NjgwMzc4OTU3ZWE1YjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GtSRyuh-fKJg4_9o8K9jRlMfJAXyTxDVbUUvmikY3uE">
> > > your solars powernet does not work
> > > just skips the SMES entirely lol
> > 
> > 
> > right so! I've fixed the solars SMES by adding a connector terminal.
> > The Solars and AME should also feed into the Main SMES line in the engine room as well (engine room is located above the service hallway) <img alt="bratan HV lines" width="1091" height="827" src="https://private-user-images.githubusercontent.com/220646896/473750490-d5b8749f-0edd-41ea-9036-c829f9a873f6.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTkzNjAsIm5iZiI6MTc1NDE5OTA2MCwicGF0aCI6Ii8yMjA2NDY4OTYvNDczNzUwNDkwLWQ1Yjg3NDlmLTBlZGQtNDFlYS05MDM2LWM4MjlmOWE4NzNmNi5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwM1QwNTMxMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NzIzZDViNGMzMjRkY2E0YjBjYmRhNzAxZDc1ZTUzYjdkNTEyMGZmNDcwZmQ1NjEyYjc5MzlkMTI1NThjMmI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QOzsjlyciSCh-0l5JWWFT6Y6seta65ELcB72HfRrA48"> <img alt="bratan solar term" width="798" height="591" src="https://private-user-images.githubusercontent.com/220646896/473750491-cfb9b43c-1a8c-4ed8-85b6-93b3a97e7f69.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQxOTkzNjAsIm5iZiI6MTc1NDE5OTA2MCwicGF0aCI6Ii8yMjA2NDY4OTYvNDczNzUwNDkxLWNmYjliNDNjLTFhOGMtNGVkOC04NWI2LTkzYjNhOTdlN2Y2OS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwM1QwNTMxMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYTllY2JhZmMyYjFhZDg0MTVhNDE0YjBhMWViOWM5NDAwODAxYWFmYTIzNmE2ZDgzZjQ1M2U3NTRhMzM1M2I2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fqgpr0ebf-uv7EgA2oNSWqQADVzp8-MQwbdv5OiozVE">
> 
> Your other smes still has two cable terminals please review your hv line
> 
> Mv line too now that I look at it

@Redrover1760 

right so, adjusted the problematic HV and MV lines

<img width="906" height="682" alt="bratan emergency termi" src="https://github.com/user-attachments/assets/fc11f598-ae56-46e5-9f58-9acc8fc5d167" />
The primary SMES having two connector terminals was intentional, it was made for an emergency generator to be anchored there, in case of the AME and solars being out for some reason. I've added a note to make this terminal's use clear for the end users

<img width="936" height="774" alt="MV line 2 + HV reroute to substation" src="https://github.com/user-attachments/assets/54a3d095-5171-4467-ae62-e012d5779129" />
Ive fixed the MV 
<img width="551" height="576" alt="MV line 1" src="https://github.com/user-attachments/assets/3052e4a0-bb23-481b-9bec-a07939199965" />
lines in the images above, added an MV line in the substation in the AME room because ive forgotten it for some reason, and ive tweaked the HV line leading to the substation on the west docking area so its connected to the SMES instead of the main HV powerlines. 

<img width="649" height="713" alt="substation line" src="https://github.com/user-attachments/assets/d55bdc7c-252c-4965-a381-b62558c4bece" />
I've kept the substation on the solar panel catwalks as is, because its like... a backup substation? just in case of catastrophic damage to the rest of the ship. It's redundant really, but its a nice touch. The LV wire there is purely to power the post light at the end.

Rubeebeebee committed:
updated it to have a roof component

Rubeebeebee commented:
updated it to have a roof component

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
added roof tiles. please work

Rubeebeebee committed:
Merge branch 'bratan-2nd-upload' of https://github.com/Rubeebeebee/Monolith into bratan-2nd-upload

Rubeebeebee committed:
updated it with starch's recommendations

Rubeebeebee commented on Resources/Prototypes/_Mono/Shipyard/bratan.yml:
```diff
@@ -0,0 +1,70 @@
+  name: PB Bratan
+  description: A mobile nightclub, cafeteria, hydroponics, and secret drug lab and smuggling ship, all in one package! Brought to you by the Paycheck Bratva. 
+  price: 142000
+  category: Medium
```
done! edited that

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> gravgen and gryos dont need to be outside, your ship has the space to have these inside, they will just get destroyed <img alt="image" width="375" height="115" src="https://private-user-images.githubusercontent.com/181524302/480093235-31d03c7e-6dc1-461d-ad83-9d5b2ba992d8.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDkzMjM1LTMxZDAzYzdlLTZkYzEtNDYxZC1hZDgzLTlkNWIyYmE5OTJkOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZjVlZWRlOWI2NWMxNmJiZjhmZmQzOGI4Mzk3MDdlNjcyZTI0NjcwYjYxMzc4ZjE2YjgzYzNiODhkNTg4Y2Y0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oIyanSKppp2GAxB4XY_loVLomt5osvP7oh4hxtibXg8">
> 
> stop thruster stacking <img alt="image" width="176" height="125" src="https://private-user-images.githubusercontent.com/181524302/480093310-2eceaef7-ee58-4466-b37d-353a2c19fc97.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDkzMzEwLTJlY2VhZWY3LWVlNTgtNDQ2Ni1iMzdkLTM1M2EyYzE5ZmM5Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kY2Y4NjM4ZGI3OWY4M2FmMjg3N2EwMDAzMWY3NzM1YTU0ZmJhM2ViY2E5ODRkNjNlYjQwOTNiZjI0ZWE0ODczJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LOFWjxISdL-dnnkboHOAOERGPDOe3rxKygoyr0T8Q_U">
> 
> dont premap guns/melee, especially not this much, and the small dim light exists for a reason <img alt="image" width="164" height="157" src="https://private-user-images.githubusercontent.com/181524302/480093514-872505b6-4938-4f77-af43-71568aebcf80.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDkzNTE0LTg3MjUwNWI2LTQ5MzgtNGY3Ny1hZjQzLTcxNTY4YWViY2Y4MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YjE2ZDZmMDU5MTg0ZDNlYjU3YThmODZjMGRhM2UxMjY0ZGEwZTI0ZGE2M2JjYTM3OTBiZThkY2NlYmFmYjU5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Nj2mww9BYpeHevjGORCEl1qIUN_5j4MkuCKJ2m8bu_I">
> 
> this lighting is obnoxious, theres a random barrel, the service techfab is in here for some reason <img alt="image" width="289" height="304" src="https://private-user-images.githubusercontent.com/181524302/480093688-d820e992-f75b-479f-8718-efb18a7727f2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDkzNjg4LWQ4MjBlOTkyLWY3NWItNDc5Zi04NzE4LWVmYjE4YTc3MjdmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YjY1MGJiZTA2OTE3N2VmMTczZGI1MjBiZjBhOGVhMGU5OGEyNjNhOGM3ZjRiNTIyMTRiM2U1NTFmODVjNWI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.qbgehZylWavEL-b5vy6kwB9mN8t2TTSrlhT616MwdOw">
> 
> flat as fuck hallway, decal up, add brick trims, etc <img alt="image" width="228" height="87" src="https://private-user-images.githubusercontent.com/181524302/480094263-e952faaf-d825-4e95-9a2c-5e389e621b09.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk0MjYzLWU5NTJmYWFmLWQ4MjUtNGU5NS05YTJjLTVlMzg5ZTYyMWIwOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZmFlY2Y5NTA4NGM1ZWNmN2VlNzFmMGM0Nzc4YzU1NjA4MTk1Y2NiN2MwMjYzMjk5OWU0ZGZmN2E3ZTIxYTMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fJCbZXjMxXmAevk2L4h1ZZqe_pDY9dLBKPL5p1uCyfk">
> 
> use freezer crate, move the vendor out of here, dont map the resupply box, and use a passive vent for the freezer instead of connecting it to distro <img alt="image" width="167" height="148" src="https://private-user-images.githubusercontent.com/181524302/480094752-cd7a81cb-516b-4a94-877a-55c0017f6f58.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk0NzUyLWNkN2E4MWNiLTUxNmItNGE5NC04NzdhLTU1YzAwMTdmNmY1OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZGUxODI1ZThlMmZlMGRhOTQ1YTc4NWNhNWU1ODViOTBjMGYyOGJmYTU4MDcxMmYxZTI1YTA1NjkyM2QxMzI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.EXsDTr4W7YKbVvo5GMZn9OmXDupHXzYEozYcLrML-8c">
> 
> wow this tiling needs a lot of work, needs chef's closet, needs drain <img alt="image" width="174" height="239" src="https://private-user-images.githubusercontent.com/181524302/480095237-1cf2651b-de36-43ef-87c4-0f1d80bb3e21.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk1MjM3LTFjZjI2NTFiLWRlMzYtNDNlZi04N2M0LTBmMWQ4MGJiM2UyMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZTY5YTBjODNjODI1ODU4NDJlYWM3ODM1YzNjNWI5Y2Q1MmRjYzc2ZGUzZjcyMTcyZDdhZDc2YTZmMDAwYmNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cMc6mhUg4g8iueCrqtTHdSH5uHf9dPFM4zjz_2OMZMU">
> 
> its a very flat room, its just dark steel tiles, like 3 decals, and a few steel tiles (and a criminal records computer??) <img alt="image" width="196" height="229" src="https://private-user-images.githubusercontent.com/181524302/480095858-4101f308-0451-465b-8afe-5d6109779be7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk1ODU4LTQxMDFmMzA4LTA0NTEtNDY1Yi04YWZlLTVkNjEwOTc3OWJlNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYzI0MDEwZjVmYmQyZGUxNGIyMTYyNWViM2JhN2YxY2JmYmU3YjEyMmU2YjAzZWQzZTU5OTBiYjQwNTg0YTdkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uDNpqPJgPqC2azL3KxUS192mTvEDkTtN64f7tzGnsnE">
> 
> these grass decals on maint tiles looks bad, this lighting looks bad, theres a random techmaint decal at the bottom, you are missing a sink and drain <img alt="image" width="132" height="238" src="https://private-user-images.githubusercontent.com/181524302/480096466-b52e7aa7-9ebc-4a96-a3cb-d9560e826057.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk2NDY2LWI1MmU3YWE3LTllYmMtNGE5Ni1hM2NiLWQ5NTYwZTgyNjA1Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZjgwZTM3MDQ1MDljMGZkMmFlOGY0MjZiMDMxOGY0MzE5OGNmMjI3ZThjMzg2ZmQxYmFkZmMzMGQyMjJkNzcyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QTO9Adg59PQTD99uLsDT6adCz6MVL8nfpRCdc3zCfhQ">
> 
> this will block the firelock <img alt="image" width="56" height="59" src="https://private-user-images.githubusercontent.com/181524302/480096585-48be7916-da1f-43aa-8d93-6dbcda983f1f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk2NTg1LTQ4YmU3OTE2LWRhMWYtNDNhYS04ZDkzLTZkYmNkYTk4M2YxZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZDk1MGNmOTAyY2JkMDNjY2ZjMDZlMTU2ODYyMzkxNDUwYmVkMmEwNDE2NGVjMWFkOTY5ZGYxNTBjMGU5ZjgxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.66BWRgSCheFi4fhuAuZFXIFV24FQgytERSdDFvoW8PA">
> 
> benches exist for a reason <img alt="image" width="127" height="128" src="https://private-user-images.githubusercontent.com/181524302/480096820-b03bb8d7-f9d9-4081-be20-aa71f0eb1285.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk2ODIwLWIwM2JiOGQ3LWY5ZDktNDA4MS1iZTIwLWFhNzFmMGViMTI4NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMGQ2OTRjYWZjNmRlZDJlNzEyMjhhNzQzY2ZhZGNiZTI5YzljYjZhZjBmNmRiYzMxMTYwYjc3MGI5N2RlNGIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.u_eeRSaN1jFEeKMiLOFKid0L3We7rUwgiX-3-8eUB20">
> 
> behold the brightest bathroom in existence for some reason?? <img alt="image" width="164" height="108" src="https://private-user-images.githubusercontent.com/181524302/480097170-75e6b1da-1786-4152-9263-ed3d0f3c65e4.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk3MTcwLTc1ZTZiMWRhLTE3ODYtNDE1Mi05MjYzLWVkM2QwZjNjNjVlNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTQ2YjIyMTY1OTdmMTBjMmE5OGQ2NDMzY2I5MzI0ODFkNDhiYTc4MDBhNWVhMWVjOTllMjhlMjg4ZWQ3ZTgxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.SAgU6dPTpnCgjZme-3Gt5mddOK45amKlmImqwaadp7o">
> 
> small lights exist for a reason <img alt="image" width="137" height="87" src="https://private-user-images.githubusercontent.com/181524302/480097319-9c3d34fa-5b7c-4b39-ba35-38c45a45d028.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk3MzE5LTljM2QzNGZhLTViN2MtNGIzOS1iYTM1LTM4YzQ1YTQ1ZDAyOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MmNiZGZhZmMyNGUxOTgzZDNhMTlmOWUyNjI1MDczODRlOTU4YTQ1NTEwMjA1ODRiZTgyM2NlZTZhYTM5NWNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rerUAXeKiE74phplJszQ5iDyHKUFE5di2446W9O7rTg">
> 
> looks flat, needs decal work, and i see that diagonal steel tile poking out horribly under the table, lighting is generally awkward and the diagonal plasma windows can be swapped out for full ones <img alt="image" width="379" height="331" src="https://private-user-images.githubusercontent.com/181524302/480097726-e97d035b-4626-41f0-ab24-7a604e9e7821.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk3NzI2LWU5N2QwMzViLTQ2MjYtNDFmMC1hYjI0LTdhNjA0ZTllNzgyMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZmQxZTY4OWJhYmNlM2IyZjgxMGUyMjU1YWIwNTBkMTFkYTc4MjllM2Q5YThlODQ3M2FmZWY5NjM5OTYwZTNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.EVZkXafWlk0QCDtSznGasH2ZOUI4Y9dWDb7CzXlGFy8">
> 
> why is it so bright? i could break into this vault by cutting a LV cable outside. <img alt="image" width="286" height="333" src="https://private-user-images.githubusercontent.com/181524302/480098076-f9dd8a50-312b-4136-9cc0-224e988f6f6c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk4MDc2LWY5ZGQ4YTUwLTMxMmItNDEzNi05Y2MwLTIyNGU5ODhmNmY2Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NmMxZWE5NTMwNzgxZWQ1NWI2ZGY1Mzc2YjRmNzNlODdkZjY3YjcyODE4OGJiMDk2NjJmOThhY2QzZGY4OWY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6m0JQvWQQRCgf9UcC0subOAAARVCSDHp0sX4UiI-eJk">
> 
> another botany and random chemlab for some reason, no drains, overly bright lighting, little decal work, and a random biosuit locker for some reason <img alt="image" width="228" height="244" src="https://private-user-images.githubusercontent.com/181524302/480098325-65a48bf5-d11b-4224-9275-c0c055e513a2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk4MzI1LTY1YTQ4YmY1LWQxMWItNDIyNC05Mjc1LWMwYzA1NWU1MTNhMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MjYxNzZjZGZmMjVmNTllMDQ2MGM4Y2IyMjE1MjMyMjRiZGExYWFhZGIyNmFkMWU1ZDQyNDA5ZTFhYWZmY2IwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xWYTL7TcRZqYyebryZIKvuHbyCyvOwJzQ8kCUVu6lUA">
> 
> the lighting is awfully bright in here, the jukebox is just in the middle of the floor, the stage barely looks elevated and has wood stairs when neither the stage itself or the ground below it is wood flooring, the mini tiles are lazily spammed with minimal decals, theres a plasma windoor instead of just a normal airlock for the bar, the reinforced glass table makes an awful counter, theres a random fucking blacklight on the wall, and a condiment dispenser for some reason instead of a sink <img alt="image" width="290" height="479" src="https://private-user-images.githubusercontent.com/181524302/480099992-7bede351-590a-4705-8c4c-8f4e08cbbf94.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMDk5OTkyLTdiZWRlMzUxLTU5MGEtNDcwNS04YzRjLThmNGUwOGNiYmY5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYTZhMjg5YzgyY2RhM2VhOGM5ZTc4MmUwNGVkMjljMTlkMTMzOWQ4YzYwMGQwNWRjYTJiYjQzZmJmMzgwN2QxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JFHjiT0PNZ-AmIiF73jbFsVIs-CtVgu2FtBemhSXfd8">
> 
> these long ass windows need some kind of wall to break them up, even if its 1 tile supports <img alt="image" width="52" height="474" src="https://private-user-images.githubusercontent.com/181524302/480100196-b05d6e57-92ef-47a5-8432-77c040ae4636.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMTAwMTk2LWIwNWQ2ZTU3LTkyZWYtNDdhNS04NDMyLTc3YzA0MGFlNDYzNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZjEwYTAyZTQ1M2EyNTAyYjVjZGYxY2NkMmJhYzc3MjMwMWNjMDk5ODM5NDUyMTBmMjE2MTQ4YThkNGUwNTNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-3-GGXUgtxAO-JXCC8Q1dOcUzt51n5UszPgLTvMIiUc">
> 
> docks need a external airlock inbetween them and rest of the hallway <img alt="image" width="154" height="82" src="https://private-user-images.githubusercontent.com/181524302/480100723-1404db44-0930-4cd0-84ab-bc72126f6fda.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MzM5OTMsIm5iZiI6MTc1NTczMzY5MywicGF0aCI6Ii8xODE1MjQzMDIvNDgwMTAwNzIzLTE0MDRkYjQ0LTA5MzAtNGNkMC04NGFiLWJjNzIxMjZmNmZkYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQyMzQ4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YTJhMzA0NWNkZGFkMjllYjM0NjY0OWZiYzI1MjUzMjdhNmY1NGNjNDQyYjA0ZmM5MGE1MTliMjU1MDgyNDJlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.i4Ob_PQ-qXjoS4pVifL5LBoDEsrtI0F-vVA1roiHWRk">

Right so! I've fixed/seen through the changes you recommended, here they are so far;

<img width="1002" height="904" alt="updated bratan profile" src="https://github.com/user-attachments/assets/e0a35ec9-31f4-46f4-980e-203ced52477e" />

for the gyroscopes + grav generator, ive removed them from the front, and placed them in a cubby by the now engineering/maintenance wing on the east

<img width="1076" height="865" alt="bratan gyroscopes" src="https://github.com/user-attachments/assets/14694b25-dcdf-4fbe-b94a-9a83d6859a0e" />
<img width="1032" height="802" alt="bratan bridge" src="https://github.com/user-attachments/assets/616a961c-ee65-486d-8398-c9522f8334e4" />

I've revammped the thrusters, reduced the number and removed the thruster stacking at the north side of the ship. It moves like a brick, as it should, and the thrusters are hopefully considered "in-built" into the hull

<img width="650" height="799" alt="bratan security + guns" src="https://github.com/user-attachments/assets/2dd5b178-1146-4b96-80c6-a91ff4e3fbef" />
<img width="556" height="818" alt="bratan security" src="https://github.com/user-attachments/assets/621bf16a-9b98-4de6-ac82-49120dab2798" />

i've removed the guns from the armory, so its just racks now, and security only has 3x disablers and 3x stun batons, and the armory has 3x baseball bats, stuff that any spacer can easily get at a non-lethal vend. I've also replaced the lights in the armory with dim ones. You really need a flashlight to see what you're doing inside now. 

<img width="641" height="692" alt="bratan service hallway" src="https://github.com/user-attachments/assets/a1f8e0c4-e754-4867-bd7c-3bf6b13936aa" />

The service hallway + maintenance halls have been darkened up, and i'd like to argue the service techfab is good as it is, its within the service hallway, where only staff is allowed anyways. 

<img width="493" height="284" alt="bratan entry 2" src="https://github.com/user-attachments/assets/05fa3b9a-1a8d-4df3-9518-86511d048267" />
<img width="418" height="317" alt="bratan entry 1" src="https://github.com/user-attachments/assets/97f65830-a936-4887-818b-cab30341d44d" />

Hallways un-flatfucked

<img width="498" height="739" alt="bratan kitchen vent" src="https://github.com/user-attachments/assets/25ef4147-23fc-4fc0-beaf-086fc2d4d119" />
renovated the kitchen and freezer, didnt have to use a passive vent, since there was a passive gate underneath the door. Should freeze the freezer, but keep distro safe. 

<img width="489" height="809" alt="bratan kitchen" src="https://github.com/user-attachments/assets/9712a5b7-e2e8-4fab-9774-2df6d949f828" />
kitchen swagged up and includes a chefs closet now. 

<img width="407" height="529" alt="bratan botany" src="https://github.com/user-attachments/assets/39da24f7-bd40-4374-bc22-7f0b24d4bbfa" />
I've fixed the textures on the garden, although i'd like to argue that the grass decals + techmaint decals are a good touch, since the grass makes it looks like the grates had seeds/plants fall over them and overtime it just grew on it, and the techmaint decals are a good seperator for the botany space and the dining room space. Like stairs. It's supposed to be like a koi-pond vibe, not a garden and not a pond. 

Also removed the offending vending machine from blocking the firelock. 

<img width="662" height="605" alt="bratan gameroom" src="https://github.com/user-attachments/assets/821c29d4-c24b-41de-a555-51be768c0a37" />

changed up the tiles in the gameroom, and added benches to the offending table. 

<img width="990" height="706" alt="bratan bottom" src="https://github.com/user-attachments/assets/65d3ae35-a63f-4eb9-b163-a5a3a6f51c50" />

the small bathroom + custodial closet is now darked as fuck, and the bottom VIP room has been improved with decals, replaced the tiles under the table with glass floors, for swag. 

<img width="421" height="467" alt="bratan vault wires" src="https://github.com/user-attachments/assets/1d5cfb08-f0e7-46b5-b2ea-f1d2fa55c66e" />
<img width="391" height="450" alt="bratan vault light" src="https://github.com/user-attachments/assets/6f97814a-f6dc-43aa-b803-13ae017de957" />

the vault has been significantly darkened up that you need a flashlight just to see inside, and the wires should be connected to other APCs so that one snip wouldnt unpower it. hooray. 

<img width="554" height="635" alt="bratan drug lab" src="https://github.com/user-attachments/assets/fc32ccf3-6cf1-4317-8344-3d64c709ae0f" />

the druglab is now darked up, and has been improved decalwise. Removed the biosuit and replaced it with a wall close to stuff the breaking bad reference yellow raincoat + gas mask


<img width="450" height="714" alt="bratan bar" src="https://github.com/user-attachments/assets/5cb04199-8394-4cab-9934-6c5b5094f547" />

I've improved the bar's decals and added a sink+drain, removed the condiment dispenser. (no more ketchup shots :( ), although i'd like to argue to keep the plasmaglass door, since a proper airlock wouldnt make a difference in case of a breach, due to the rest of the bar being open air. I'd also like to argue that the reinforced glass table is perfect for bar tables, since its supposed to be a nightclub of some sorts. I've removed the blacklights and just kept the pink/blue lights for the vibes. The stage is supposed to be plain, since ive provided a crowbar + tile cutter for the end user to make the stage whatever theme they like. 

<img width="302" height="832" alt="bratan window 2" src="https://github.com/user-attachments/assets/15988e40-484e-472e-8007-2f22440751b9" />
<img width="372" height="834" alt="bratan window 1" src="https://github.com/user-attachments/assets/85b017b3-b6bc-4ed4-b21a-004fd54ea9a3" />

broke up the windows with reinforced walls, as requested

<img width="963" height="522" alt="bratan extermals 2" src="https://github.com/user-attachments/assets/46bae5d5-66f2-4faa-810f-348f5204f0f1" />
<img width="917" height="586" alt="bratan externals 1" src="https://github.com/user-attachments/assets/29ca37ad-2106-405f-b3f3-298bf97bb2c8" />

added external airlocks leading up to the external docking airlocks. They're still able to be shuttered, since ive used the glass variants. 



@starch70 you may have reviewed this ship using outdated photos, but i think thats on me for not transfering the updated photos from the previous upload.

Avalon-Proto commented:
That AME looks so silly tacked on like that

Rubeebeebee commented:
> That AME looks so silly tacked on like that

It does give a vibe of 

"FUCK we need to make this station move- but we need more power."
" Slap an AME on one of the docking bays"
"Sir gyroscope and grav gen wont fit!"
"Put it in sideways"

Rubeebeebee commented:
Yippie!! @starch70 thank you!!
